packages/cli/src/commands/start.ts

```diff
@@ -221,8 +221,11 @@ export class Start extends BaseCommand {
 		this.logger.debug('External secrets init complete');
 		this.initWorkflowHistory();
 		this.logger.debug('Workflow history init complete');
-		await this.initTestRunner();
-		this.logger.debug('Test runner init complete');
+
+		if (!isMultiMainEnabled) {
```

Comment:
How do we plan to do this in multi-main? Do you have a follow-up ticket?

Reply:
We track known limitations / tech debt like this in Notion for now. I'm not sure how to deal with it in multi-main setup yet, need to get familiar with the execution recovery service first. I guess the logic would be similar, or maybe we can even re-use part of existing code.